src/org/exist/launcher/LauncherWrapper.java

```diff
@@ -59,45 +66,50 @@ public LauncherWrapper(String command) {
     }
 
     public void launch() {
-        launch(true);
+        final String home = System.getProperty("exist.home", ".");
+        final PropertiesConfiguration vmProperties = getVMProperties();
+
+        final OperatingSystem os = OperatingSystem.instance();
+        final ProcessManager pm = os.processManagerInstance();
+        final Process process = pm.createProcess();
+        final String cmdLine = getJavaCmd() + getJavaOpts(home, vmProperties) + " -jar start.jar " + command;
+        process.setWorkingDir(home);
+        process.setVisible(false);
+        process.setPipeStreams(false, false);
+        process.setCommand(cmdLine);
+        System.out.println(cmdLine);
```

Comment:
yes, I could not test every system, so it's good to have the actual command line printed out.